json_serializable/pubspec.yaml

```diff
@@ -36,3 +36,7 @@ dev_dependencies:
   test_descriptor: ^2.0.0
   test_process: ^2.0.0
   yaml: ^3.0.0
+
+dependency_overrides:
```

Comment:
Need to bump the version range on `json_annotation` to be "tight" w/ what you change in the json_annotation package

Reply:
also changelog!